examples/let-expression/let_expression.bal

```diff
@@ -0,0 +1,15 @@
+import ballerina/io;
```

Comment:
Why don't we have any comment for the example? That won't look good when rendering the BBE in b.io

Reply:
This is just a draft PR, to get some ideas on what to include in the BBE (using let expressions in different scenarios)

I will make sure to follow the guidelines and practices we follow on bb.io once this PR is ready to be merged.